### DISCUSSION TITLE
split files for each network protocol?

### DISCUSSION THREAD

**Discussion by ghost** [filecxx/FileCentipede#798] [Q&A] [answered]

**Comment by filecxx** [comment]
Could you please provide an example for better understanding?

**Reply by ghost** [reply]
Suppose you have one file on an ftp server, the other file on an https server. How do I recover the file in two different places?

**Reply by filecxx** [reply, marked as answer]
> Suppose you have one file on an ftp server, the other file on an https server. How do I recover the file in two different places?

Are the two shared files identical in content? Are you aiming to download the file from two distinct locations and protocols to speed up the process, or do you simply need to download two different files? If it's the latter, you'll need to use separate addresses for each file download. For example:

'''https://127.0.0.1/1.txt'''
'''ftp://127.0.0.1/1.txt'''

**Reply by ghost** [reply]
> Are the two shared files identical in content?

no.

> Are you aiming to download the file from two distinct locations and protocols to speed up the process, 

yes. What do I do here?
